modules/test-utils/src/browser-automation/browser-driver.ts

```diff
@@ -96,6 +98,10 @@ export default class BrowserDriver {
     }
     await Promise.all(promises);
 
+    if (coverage) {
+      await this.page.coverage.startJSCoverage();
```

Comment:
A few more comments maybe? Are these methods part of puppeteer or installed by the dependency?